openhcl/hcl/src/ioctl.rs

```diff
@@ -583,13 +583,13 @@ pub struct MshvVtlLow {
 impl MshvVtlLow {
     /// Opens the device.
     pub fn new() -> Result<Self, Error> {
-        let file = std::fs::OpenOptions::new()
```

Comment:
Should we add these to our clippy restriction list, and force fs-err everywhere?

Reply:
I don't think so.

Reply:
Why not?